pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -212,6 +212,8 @@ let
       MPTCP           = whenAtLeast "5.6" yes;
       MPTCP_IPV6      = whenAtLeast "5.6" yes;
       INET_MPTCP_DIAG = whenAtLeast "5.9" (mkDefault module);
+    } // optionalAttrs (stdenv.hostPlatform.system == "aarch64-linux") {
+      TCP_CONG_ADVANCED = yes;
```

Comment:
Can you move this into the main attrset and write it like this:

```
TCP_CONG_ADVANCED = mkIf (stdenv.hostPlatform.system == "aarch64-linux") yes;
```

Adding a comment as to why it's needed would be good. You can imagine someone passing by and thinking it's _only_ being enabled for aarch64, which isn't the case (given the default behavior for x86_64)

Reply:
What about removing the condition like @samueldr [said](https://github.com/NixOS/nixpkgs/pull/144641#issuecomment-961312482)?